packages/server/lib/routes.ts

```diff
@@ -107,6 +108,7 @@ const upload = multer({ storage: multer.memoryStorage() });
 router.get('/health', (_, res) => {
     res.status(200).send({ result: 'ok' });
 });
+router.get('/env.js', getEnvJs);
```

Comment:
should we have the extension in the route path or not? I believe this is the only endpoint we are doing so

Reply:
it's returning a javascript file not a json, so I wanted to be sure it's was highlighted, wdyt

Reply:
I don't feel strongly either way, just wanted to make sure it was intended.